helmfile/charts/vulnerability-exporter/values.yaml

```
@@ -0,0 +1,2 @@
+vulnerabilityExporter:
+  schedule: "*/1 * * * *"
```

```suggestion
  schedule: "* * * * *"
```
Nit: this should be the same, right?